spec/fixtures/Podfile_UsesFrameworksBoolean.yaml

```diff
@@ -0,0 +1,15 @@
+---
+installation_method:
+  name: cocoapods
+sources:
+  - https://cdn.cocoapods.org/
+target_definitions:
+  - abstract: true
+    children:
+      - children:
+          - abstract: false
+            inheritance: search_paths
+            name: SampleAppTests
+        name: SampleApp
+        uses_frameworks: true
```

Comment:
this is used for a test to ensure we can parse a pre-existing serialized `YAML` `Podfile` with a boolean value for `uses_frameworks`.